lib/ButtonNew/ButtonIcon/ButtonIconDanger.js

```diff
@@ -52,7 +52,7 @@ function ButtonIconDanger({
 
   return (
     <ButtonBase className={classes} disabled={disabled} size={false} {...rest}>
-      <Icon name={name} types={iconTypes} />
+      <Icon name={name} types={iconTypes} defaultActiveColor={false} />
```

Comment:
Icon gets stuck as blue, even though it should be red when used as a dropdown trigger without this.